pkg/apis/componentconfig/types.go

```diff
@@ -610,6 +610,16 @@ type KubeControllerManagerConfiguration struct {
 	// concurrentGCSyncs is the number of garbage collector workers that are
 	// allowed to sync concurrently.
 	ConcurrentGCSyncs int32 `json:"concurrentGCSyncs"`
+	// nodeEvictionRate is the number of Node evictions NodeController does in ordinary circumstances
+	NodeEvictionRate float32 `json:"nodeEvictionRate"`
+	// secondaryNodeEvictionRate is the number of Node evictions NodeController does when zone is unhealty
+	SecondaryNodeEvictionRate float32 `json:"secondaryNodeEvictionRate"`
+	// largeClusterSizeThreshold is the number of Nodes from which NodeController treats a cluster as "large"
```

Comment:
I would replace this comment with
secondaryNodeEvictionRate is implicitly overridden to 0 for clusters smaller than largeClusterSizeThreshold